charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -58,6 +58,12 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--v={{ .Values.logLevel }}"
             - "--drivername=$(DRIVER_NAME)"
+{{- if .Values.instanceID }}
+            - "--instanceid={{ .Values.instanceID }}"
+{{- end }}
```

Comment:
@zerotens 
Can you please open another for instanceID change ?

Reply:
@Rakshith-R 
I made an extra PR for this.

https://github.com/ceph/ceph-csi/pull/4666